python/ql/lib/semmle/python/frameworks/Streamlit.qll

```diff
@@ -8,6 +8,7 @@ import semmle.python.dataflow.new.RemoteFlowSources
 import semmle.python.dataflow.new.TaintTracking
 import semmle.python.ApiGraphs
 import semmle.python.Concepts
+import semmle.python.frameworks.SqlAlchemy
```

Comment:
```suggestion
private import semmle.python.frameworks.SqlAlchemy
```
make this private to remove the warning (we are not interested in re-exporting it anyway).